share/python/driver.py

```diff
@@ -442,7 +442,7 @@ def energy(name, **kwargs):
     wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
 
     for postcallback in hooks['energy']['post']:
-        postcallback(lowername, **kwargs)
+        postcallback(lowername, wfn=wfn, **kwargs)
 
```

Comment:
Ah, the C-side pre/postcallback. Yes, I agree. I think @jturney put those in to plot orbitals in the course of SCF interations. But in a py-driven SCF, could probably remove c-side and use wfn to do the task.

Good catch on the EFP. Basically all user-facing fns need the below as initial lines. Then kill off `lowername`. Agree?

```py
kwargs = p4util.kwargs_lower(kwargs)
if not hasattr(name, '__call__'):
    name = name.lower()
```